### ISSUE TITLE
The `set-output` command is deprecated and will be disabled soon

### ISSUE THREAD

**Issue by gtazzoli-uala** [terraform-docs/gh-actions#97]
### Describe the bug

When running from @v1.0.0 or main I get:

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: 

### How can we reproduce it?

      - name: Update README.md docs
        id: update-readme
        uses: terraform-docs/gh-actions@main #or v1.0.0 
        with:
          working-dir: .
          output-file: README.md
          output-method: inject
          git-push: "true"
          config-file: './.config/.terraform-docs.yml'

**Comment by lbeltramino** [comment]
I'm having the same issue. Hope this is fixed soon.

**Comment by cordobacancun** [comment]
Same Here! Hope this is fixed soon.

**Comment by charly942** [comment]
I still have the same problem too, hope this is fixed soon.

**Comment by jhegert-infraDevOps** [comment]
Experiencing the same issue. Hope they fix this soon!

**Comment by GonzaloRodriguezCabrera** [comment]
I'm having the same issue. Hope this is fixed soon.

**Comment by heuerluis** [comment]
Hi all, I´ve got the same problem. Thanks so much for showing this issue. Fixing it will be great for me too.

**Comment by jvidelaolmos** [comment]
I'm facing the same issue here.

**Comment by gomeznf** [comment]
Hi, I'm having the same issue here. Hope this is fixed soon. Thanks in advance.

**Comment by db4rr3t0** [comment]
I'm having the same issue here.

**Comment by katanita** [comment]
I'm facing the same issue here. Hope it gets fixed soon.

**Comment by nicolascavedale** [comment]
I'm having the same issue! Please fix this.

**Comment by jonathanio** [comment]
There is a commit (https://github.com/terraform-docs/gh-actions/commit/18dc76d9b2e3c746cf6f8e073c7fa7df16dcf620) with the fix in `main`, but there is no release with this change. Getting a `v1.0.1` release out would be great!

I think in the meantime you can use the following override:

```yaml
    - name: Render terraform docs inside the README.md and push changes back to PR branch
      uses: terraform-docs/gh-actions@18dc76d9b2e3c746cf6f8e073c7fa7df16dcf620
      with:
        ...
```

**Comment by ipc-zpg** [comment]
even with this:

```
       - name: Render terraform docs and push changes back to PR
         uses: terraform-docs/gh-actions@18dc76d9b2e3c746cf6f8e073c7fa7df16dcf620
         with:
           working-dir: ./liftshift_terraform/
           output-file: README.md
           output-method: inject
           git-push: "true"
```

I am getting:

```
<!-- END_TF_DOCS --> ./liftshift_terraform/
liftshift_terraform/README.md updated successfully
::debug No change in ./liftshift_terraform// detected
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
::debug No files changed, skipping commit
```

**Comment by gtazzoli-uala** [comment]
Hello,

Just an update. From @main still the same. 
What I did was to create my own Docker image using the Dockerfile that was updated on recent commits and pulled that image to my own public repository. Then I forked this repo and updated the action.yml with the new image and the warnings have gone. 
They need to build and pull a new image and update the action.yml on this repo to make it work. Until that we have to wait or make our own.

Cheers!
Gus.

**Comment by chrisdleech-pol** [comment]
> Hello,
> 
> Just an update. From @main still the same. What I did was to create my own Docker image using the Dockerfile that was updated on recent commits and pulled that image to my own public repository. Then I forked this repo and updated the action.yml with the new image and the warnings have gone. They need to build and pull a new image and update the action.yml on this repo to make it work. Until that we have to wait or make our own.
> 
> Cheers! Gus.

Thanks for the tip, it worked for me. I had to add the following to the Dockerfile to get it to build though: https://www.alpinelinux.org/posts/Alpine-edge-signing-keys-rotated.html

**Comment by marcofranssen** [comment]
Could we get a release containing the fix mentioned above?

**Comment by arogic** [comment]
Can we have an ETA? Thanks in advance.

**Comment by marcofranssen** [comment]
@metmajer are you able to help cutting a release?

**Comment by gonzaatcroud** [comment]
@metmajer @khos2ow appreciate the help if possible, I'd hate to have to remove the action from our workflows given it's been working so well.

**Comment by bahag-hammera** [comment]
Are there any news on this topic?

**Comment by sangameshajm** [comment]
GitHub is monitoring telemetry for the usage of these `save-state` and `set-output` and plan to fully disable them on 31st May 2023. Starting 1st June 2023 workflows using `save-state` or `set-output` commands via stdout will fail with an error.

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Please try to address this asap

**Comment by MarkRoss-Eviden** [comment]
Any update on this, is this repo and action still being managed?

**Comment by marcofranssen** [comment]
@metmajer @khos2ow is this repo still maintained? Are you looking for maintainers? Would be great to get some updates on this action.

**Comment by khos2ow** [comment]
This is now released and available in [v1.1.0](https://github.com/terraform-docs/gh-actions/releases/tag/v1.1.0). Sorry for the very long delay.

I'm going to close this and mark it as done.
